transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
It's the docstring for the whole module, which includes DNS related objects. Not sure what else to put there.